mesop/components/input/input.py

```diff
@@ -14,15 +14,28 @@
 
 @dataclass(kw_only=True)
 class EnterEvent(MesopEvent):
-  """Represents an "Enter" keyboard event."""
+  """[Deprecated] Represents an "Enter" keyboard event. Use InputEnterEvent instead."""
```

Comment:
I see that this is deprecated. But seems like you wouldn't be able to actually use it? I thought if it was deprecated, the event would still be usable?  Or maybe I'm misreading the code.

Reply:
It's kinda confusing. You can use the EnterEvent, but there's no reason to, and you should use the InputEnterEvent instead. The main reason why I left it here is that if I removed it, it would be a breaking change. I think the hope is that people will migrate from EnterEvent to InputEnterEvent and then we can get rid of EnterEvent in a future version of Mesop (in hindsight, it was a mistake to have an event named EnterEvent).